scripts/clang-format.py

```diff
@@ -14,7 +14,7 @@
     'lib',
     'master',
 ]
-husky_root = os.getenv('HUSKY_ROOT', '.')
+husky_root = '.';
```

Comment:
Yes, I know that. I thought normally users won't run the scripts in another path.